### PR TITLE
Fix bug with telescope indices

### LINKED ISSUE #84: Version 0.2.0 benchmarks can not reproduced with current master using standard config file
I had problems reproducing the benchmarks (version 0.2.0; single tel LST) with the current master using a standard config file (see above). Bryan runs into the same problem on shevek. The run finish after ~20 mins/15000 step with accuracy (LST single_tel): INFO:tensorflow:Saving dict for global step 14955: accuracy = 0.59607786, accuracy_gamma = 0.3556767, accuracy_proton = 0.78488666, auc = 0.59984684, global_step = 14955, loss = 1.3337153 

config parameters:
Data:
  Input:
    batch_size: 64
    map: true
    num_parallel_calls: 2
    prefetch: true
    prefetch_buffer_size: 1
    shuffle: true
    shuffle_buffer_size: 10000
  Loading:
    example_type: single_tel
    min_num_tels: 1
    seed: 1234
    selected_tel_types: ['LST:LSTCam']
    selected_tel_ids:
      'LST:LSTCam': [1, 2, 3, 4]
    validation_split: 0.1
  Processing:
    sorting: null
  file_list: /home/tjark/prototype_files_class_balanced.txt
  format: HDF5
Image Mapping:
  hex_conversion_algorithm: 'oversampling'
  padding:
    LSTCam: 2
    FlashCam: 1
    NectarCam: 2
    SCTCam: 4
    DigiCam: 1
    ASTRICam: 0
    CHEC: 0
    VERITAS: 1
Logging:
  model_directory: /home/tjark/logs/hex_conversion_algorithm_test/oversampling/LST_single_tel_config
Model:
  Model Parameters:
    basic:
      conv_block:
        batchnorm: false
        bottleneck: null
        layers:
        - filters: 32
          kernel_size: 3
        - filters: 32
          kernel_size: 3
        - filters: 64
          kernel_size: 3
        - filters: 128
          kernel_size: 3
        max_pool:
          size: 2
          strides: 2
    cnn_rnn:
      cnn_block:
        function: conv_block
        module: basic
      dropout_rate: 0.5
      pretrained_weights: null
    single_tel:
      network:
        function: conv_block
        module: basic
      pretrained_weights: null
  model:
    function: single_tel_model
    module: single_tel
  model_directory: /home/tjark/ctlearn/ctlearn/default_models
Prediction:
  export_as_file: true
  prediction_file_path: /home/tjark/predictions/predictions.csv/prediction_LST_single_tel_config.csv
  true_labels_given: true
TensorFlow:
  run_TFDBG: false
Training:
  Hyperparameters:
    adam_epsilon: 1.0e-08
    apply_class_weights: true
    base_learning_rate: 5.0e-05
    optimizer: Adam
    scale_learning_rate: false
    variables_to_train: null
  num_training_steps_per_validation: 2500
  num_validations: 15

### PR DESCRIPTION
**Pull request by aribrill**
When appending ted ids to the list of triggered telescopes, the index used was mistakenly the telescope number index (e.g. 0-3 for LSTs), not the image index in the file, causing the incorrect telescopes to be read.

Closes #84.

I tested this fix with a single tel LST training run using the config settings listed in #84 and got validation accuracy 69.50% and AUC 0.7767, very close to the v0.2.0 benchmarks.

### CODE REVIEW

**Comment by TjarkMiener** [comment]
I can confirm the numbers above.

INFO:tensorflow:Saving dict for global step 43482: accuracy = 0.70417595, accuracy_gamma = 0.80503565, accuracy_proton = 0.6034061, auc = 0.7925084
